src/docs/the-new-sharetribe/the-new-sharetribe-tutorial/tutorial-marketplace-layout/index.md

```diff
@@ -32,7 +32,7 @@ option.
 Image carousel with thumbnails is a good default for most marketplaces.
 However, if the listings in your marketplace are properties with
 attractive images, screen-wide cover photos can make the listing page
-look more attractive. Our saunas have such beautiful images, so we're
+look more attractive. Our bikes have such good looking images that we're
```

Comment:
Could I suggest either
- using the carousel layout, or
- rewording the "properties" reference to something else (as "properties" suggest real estate, which we're no longer working with)

Reply:
Synced IRL, let's use carousel layout!